src/logout.js

```diff
@@ -25,6 +32,24 @@ export async function logout({ redirect } = {}) {
   }
 }
 
+async function completeSamlLogout() {
+  if (!store.tokens.accessToken) return;
+
+  try {
+    const { data } = await axios.get(`${store.baseUrl}auth/saml/idp/token`, {
+      headers: {
+        authorization: `Bearer ${store.tokens.accessToken}`,
+      },
+    });
+
+    window.location.assign(
+      `${store.baseUrl}auth/saml/idp/logout?tenant_id=${store.tenantId}&token=${data.token}&uuid=${store.user.userUuid}`
+    );
+  } catch (error) {
+    throwFormattedError(error);
+  }
```

Comment:
Might be nice to have a check for `data.token` after the get request and throw a descriptive error if not present. As it stands, we can add that later though.